### DISCUSSION TITLE
Comunication between rassbery pi 4B and spike prime or inventor hub

### DISCUSSION THREAD

**Discussion by DexterTaha** [pybricks/support#1655] [General]
there is any method to connect  spike prime or inventor hub to s rassbery pi 4B using wireless or wire comunication?
i try bleak but is not sported by raspberry os

**Comment by dlech** [comment]
Can you describe more about how you want to use this? Are use using the RPi to program the SPIKE hub or do you want to use a different computer to program it and communicate with the RPi while the program is running?

**Reply by DexterTaha** [reply]
i use rassbery pi 4b in iot project with some sensor to detect some cases and send it to the lego hub that have an if statement to do a specific things depending on the data (type of data is int) i try bleak before and now i search to make Bluetooth low energy comunication in VS code using Python or some thing related to that

**Comment by dlech** [comment]
You might be interested in https://github.com/orgs/pybricks/discussions/1599#discussioncomment-9718724

**Comment by Ibrahim-Alsheikh** [comment]
You can try this method https://github.com/Ibrahim-Alsheikh/LEGO-Spike-USB-Serial-Console
